apps/antalmanac/src/stores/Schedules.ts

```diff
@@ -134,10 +145,11 @@ export class Schedules {
     }
 
     /**
+     * @deprecated This method is no longer used by any features
      * Append all courses from current schedule to the schedule with the target index.
      * @param to Index of the schedule to append courses to. If equal to number of schedules, will append courses to all schedules.
      */
-    copySchedule(to: number) {
+    copyToSchedule(to: number) {
```

Comment:
question: If this is no longer being used, can we just remove it?

Reply:
sure; originally i left it in case we ever wanted to implement something that needed schedule copying using index but on second thought nothing else would really need that (other than the feature we took out).